modules/serverless-global-config-broker-class-default.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+//  * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-global-config-broker-class-default_{context}"]
+= Configuring the default broker class
+
+You can use the `config-br-defaults` config map to specify default broker class settings for Knative Eventing. You can specify the default broker class for the entire cluster or for one or more namespaces. Currently the channel-based broker (`MTChannelBasedBroker`) and Kafka broker (`KafkaBroker`) types are supported.
+
+.Prerequisites
+
+* You have administrator permissions on {product-title}.
+* You have installed the {ServerlessOperatorName} and Knative Eventing on your cluster.
+* If you want to use Kafka broker as the default broker implementation, you must also install the `KnativeKafka` CR on your cluster.
+
+.Procedure
+
+* Modify the `KnativeEventing` custom resource to add configuration details for the `config-br-defaults` config map:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  defaultBrokerClass: Kafka <1>
+  config: <2>
+    config-br-defaults: <3>
+      default-br-config: |
+        clusterDefault: <4>
+          brokerClass: KafkaBroker
```

Comment:
it is `Kafka`: https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/test/config/config-br-defaults.yaml#L25